extensions/grpc/runtime/src/main/java/io/quarkus/grpc/runtime/supports/context/GrpcDuplicatedContextGrpcInterceptor.java

```diff
@@ -100,6 +100,7 @@ private void close(Throwable t) {
                 Throwable nt = ehp.transform(t);
                 StatusException sre = (StatusException) ExceptionHandlerProvider.toStatusException(nt, false);
                 Optional<Metadata> metadata = ExceptionHandlerProvider.toTrailers(nt);
+                log.warn("Closing gRPC call ...", t);
```

Comment:
Is that one really useful? I would reduce the level to "debug". Also, we should mention that the "close" is because of a failure.

Reply:
Why debug?
This shouldn't happen often ... and it's a failure ... hence `warn` ...

Yeah, I'll add something about failure.